java/build/src/main/java/sleeper/build/maven/ShowInternalDependencies.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.build.maven;
+
+import sleeper.build.chunks.ProjectStructure;
+import sleeper.build.dependencydraw.DrawDependencyGraph;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+class ShowInternalDependencies {
```

Comment:
I think this class should probably be public. This is also the class that needs a private constructor according to Checkstyle. It's a convention that just avoids anyone creating an instance of the class when there's no benefit to doing that, since all its methods are static.